mpd/mpd.go

```diff
@@ -19,6 +19,17 @@ const (
 	DASH_PROFILE_LIVE DashProfile = "urn:mpeg:dash:profile:isoff-live:2011"
 	// On Demand Profile
 	DASH_PROFILE_ONDEMAND DashProfile = "urn:mpeg:dash:profile:isoff-on-demand:2011"
+	// HbbTV Profile
+	DASH_PROFILE_HBBTV DashProfile = "urn:hbbtv:dash:profile:isoff-live:2012"
```

Comment:
Given this changes for HBBTV 2.0 onwards, should we name the constant `DASH_PROFILE_HBBTV_1_5`?
Should it also contain the fast that its a Live Profile? Maybe `DASH_PROFILE_HBBTV_1_5_LIVE` ?